sim/hunter/item_sets_pve_phase_8.go

```diff
@@ -0,0 +1,216 @@
+package hunter
+
+import (
+	"time"
+
+	"github.com/wowsims/sod/sim/core"
+	"github.com/wowsims/sod/sim/core/proto"
+	"github.com/wowsims/sod/sim/core/stats"
+)
+
+var ItemSetDawnstalkerProwess = core.NewItemSet(core.ItemSet{
```

Comment:
The sets don't exist yet in the database so you have to comment out the `core.NewItemSet` calls, both this one and the ranged one (like I did in sim/paladin/item_sets_pve_phase_8.go).
We can uncomment them again after running proper db scrapes